### ISSUE TITLE
ALL my actions were failing with intialize for SPFxEmailTracker Logic app

### ISSUE THREAD

**Issue by vijayram1260** [Azure/LogicAppsUX#6192]
### Describe the Bug with repro steps

![Image](https://github.com/user-attachments/assets/1b464fb8-7bf3-4651-a998-06ac48b2f6a2)

### What type of Logic App Is this happening in?

Consumption (Portal)

### Which operating system are you using?

Windows

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Workflow JSON

```json

```

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/42dd13e7-e700-4f04-9cac-c2967b613411)

### Browser

Tried on Chrome browser

**Comment by ccastrotrejo** [comment]
Hi @vijayram1260 thanks for raising this issue. This issues looks quite similar to one that we had a couple of weeks ago.
- Is there any chance we can get your logic app schema?
- Did this logic app used to work seamlessly in the old designer?
- Do you have app settings parameterization within your connections?
- Could you try opening your logic app in the following private url? https://aka.ms/emastaging This private url, might have the fix for it.

**Comment by vijayram1260** [comment]
Yes, In old design it works perfectly. Now I was unable to switch to old
design as well.

        "$schema": "
https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#
",
        "contentVersion": "1.0.0.0",

Regards,
--------------------------------------------------------------------------------
Vijay Ram.T

**Comment by ccastrotrejo** [comment]
I got the customer workflow schema and saw that the subscription id from the connection’s `id` and `connectionId` properties weren't matching. In the old designer, this was supported because we used to extract the id from the second property to make the API call. But for new designer this should match
